src/mode/modeHandler.ts

```diff
@@ -129,50 +129,63 @@ export class VimSettings {
   useCtrlKeys     = false;
 }
 
+export enum SearchDirection {
```

Comment:
Big thumbs up on making this an enum. There's no need to retain my old 1 and -1. The default values are fine.